src/test/scala/org/broadinstitute/dsde/firecloud/service/NihApiServiceSpec.scala

```diff
@@ -73,8 +73,9 @@ class NihApiServiceSpec extends ApiServiceSpec {
     Get("/nih/status") ~> dummyUserIdHeaders(services.thurloeDao.linkedButExpiredUserId) ~> sealRoute(services.nihRoutes) ~> check {
       status should equal(OK)
       val linkExpireTime = responseAs[NihStatus].linkExpireTime.get
-      //gives a 60 second window to make sure that the link was updated to *approximately* now + 30 days
-      assert(DateUtils.nowPlus30Days - 60000L <= linkExpireTime && linkExpireTime <= DateUtils.nowPlus30Days)
```

Comment:
since `linkExpireTime` is in seconds, not milliseconds (it's divided by 1000 before it's saved), this test wasn't exactly testing what I had thought and seemed to be sporadically failing

the new assert statements are more explicit, more correct, and hopefully more reliable